activiti-cloud-services-runtime-bundle/activiti-cloud-services-subscriptions/src/main/java/org/activiti/services/subscriptions/behavior/BroadcastSignalEventActivityBehavior.java

```diff
@@ -0,0 +1,43 @@
+package org.activiti.services.subscriptions.behavior;
+
+import org.activiti.bpmn.model.Signal;
+import org.activiti.bpmn.model.SignalEventDefinition;
+import org.activiti.cloud.services.api.commands.SignalCmd;
+import org.activiti.engine.delegate.DelegateExecution;
+import org.activiti.engine.delegate.Expression;
+import org.activiti.engine.impl.bpmn.behavior.IntermediateThrowSignalEventActivityBehavior;
+import org.activiti.engine.impl.context.Context;
+import org.activiti.engine.impl.interceptor.CommandContext;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Scope("prototype")
+@Component("defaultThrowSignalEventBehavior")
```

Comment:
@daisuke-yoshimoto let's use a constant in the same way that we are using in the ServiceTaskBehaviour. This also needs to be reflected in the Activiti Engine project right? Meaning that you need to extend the mechanism to pick up this behaviour in cloud environments.

Reply:
@Salaboy 
If it is in CloudActivityBehaviorFactory.class, I can define a constant.
Activiti Cloud's constants can not be defined in Activiti engine because dependencies are reversed.

Reply:
Of course, but we should follow the same approach as for the ServiceTaskBehaviour.. I will find the links and have a chat with you in gitter.